### PR TITLE
Send file path to the frontend

### PR DESCRIPTION
**Pull request by oeway**
This PR send the file path to the frontend.
For security reason, elfinder do not expose any real file path to the frontend (only hash values). However, it does not necessary for working with jupyter servers which are already protected with a token, and an entire VM is exposed.

### CODE REVIEW

File: jupyter_elfinder/views.py [hunk 1]
```diff
@@ -80,8 +80,9 @@ def connector(request: Request) -> Response:
         "url": request.registry.settings["files_url"],
         "base_url": request.registry.settings["base_url"],
         "upload_max_size": 100 * 1024 * 1024 * 1024,  # 100GB
-        "debug": True,
         "tmb_dir": request.registry.settings.get("jupyter_elfinder_thumbnail_dir"),
+        "send_file_path": request.registry.settings["send_file_path"],
+        "debug": True,
```
Comment (MartinHjelmare) [inline jupyter_elfinder/views.py]: Should we add a debug option in the cli arguments? Can be another PR.

Comment (oeway) [reply jupyter_elfinder/views.py]: Good idea. Let's do that in another PR.

**Comment by oeway** [comment]
Close this one, since these changes has already been made in https://github.com/imjoy-team/jupyter-elfinder/pull/50.
